### DISCUSSION TITLE
Laravel IntelliDB | Intelligent way to generate database-related laravel "artisan make" components using OpenAI

### DISCUSSION THREAD

**Discussion by salehhashemi1992** [HassanZahirnia/laravel-package-ocean#96] [Package Suggestions]
A Laravel package that provides an intelligent way to generate database-related components using OpenAI.

It extends the default artisan make commands in Laravel to automatically generate the content of each component using AI, based on the provided description.

Repo: https://github.com/salehhashemi1992/laravel-intelli-db
Packagist: https://packagist.org/packages/salehhashemi/laravel-intelli-db

**Comment by salehhashemi1992** [comment]
Showcase video: https://www.youtube.com/watch?v=AvKUazH7l5U
And: https://www.youtube.com/watch?v=hwtFBxdnq6o

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!

**Reply by salehhashemi1992** [reply]
Thank you. This package is written by me and I hope my other packages will be on the list soon 👌
